### PR TITLE
UI/stripe connect payment settings

### PR DESCRIPTION
**Pull request by Shruti-Apte**
## Notion card
https://www.notion.so/saeloun/Stripe-Connect-for-all-accounts-4fd1069594564038b6f2b39f0fcf0bd5
## Summary

payment settings page UI 
## Preview

paymentSettingUpdated

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/views/payments/index.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div class="max-w-6xl mx-auto px-2 md:px-11 font-manrope">
   <div>
     <div>
-      <%= react_component("Payments", {
+      <%= react_component("payments", {
```
Comment (ajinkyaa) [inline app/views/payments/index.html.erb:4]: Component name should always start with Capital letter

File: app/views/payment_settings/index.html.erb [hunk 2]
```diff
@@ -1 +1,9 @@
-<H1>Code</H1>
+<div class="max-w-6xl mx-auto px-2 md:px-11 font-manrope">
+  <div>
+    <div>
+      <%= react_component("payment_settings", {
```
Comment (ajinkyaa) [inline app/views/payment_settings/index.html.erb:4]: Component name should always start with Capital letter

**Comment by supriya3105** [comment]
@Shruti-Apte Can you share loom video for this

**Comment by apoorv1316** [comment]
@Shruti-Apte Loom video for this please

**Comment by Shruti-Apte** [comment]
> @Shruti-Apte Loom video for this please

**Review by apoorv1316 — APPROVED** [review]

**Pull request merged**